File: nova-care-web/src/types/bodyRegion.ts
export type Side = 'center' | 'left' | 'right';
export type BodyView = 'front' | 'back' | 'both';
export type BodyCategory = 'head_neck' | 'trunk' | 'musculoskeletal' | 'limbs';

export interface BodyRegion {
  id: string;
  name: string;
  side: Side;
  view: BodyView;
  category: BodyCategory;
}

File: mobile/app/src/main/java/com/example/kltn_novacare/data/model/BodyRegion.kt
package com.example.kltn_novacare.data.model

data class BodyRegion(
    val id: String,
    val name: String,
    val displayName: String,
    val side: String, // "left", "right", "center"
    val category: String, // "head", "neck", "chest", "abdomen", "back", "shoulder", "arm", "leg"
    val modelEntityNames: List<String> = emptyList()
)
